scripts/build

```diff
@@ -1,9 +1,33 @@
+#!/bin/bash
 set -e
 
 source $(dirname $0)/version
 
 cd $(dirname $0)/..
 
+declare -A OS_ARCH_ARG
+
+OS_PLATFORM_ARG=(linux windows)
+OS_ARCH_ARG[linux]="amd64"
+OS_ARCH_ARG[windows]="amd64"
+
 mkdir -p bin
 [ "$(uname)" != "Darwin" ] && LINKFLAGS="-linkmode external -extldflags -static -s"
 CGO_ENABLED=0 go build -ldflags "-X main.VERSION=$VERSION $LINKFLAGS" -o bin/rancher-dns
+
+if [ -n "$CROSS" ]; then
+    rm -rf build/bin
+    mkdir -p build/bin
+    for OS in ${OS_PLATFORM_ARG[@]}; do
+        for ARCH in ${OS_ARCH_ARG[${OS}]}; do
+            OUTPUT_BIN="bin/$OS/$ARCH/rancher-dns"
+            if test "$OS" = "windows"; then
+                OUTPUT_BIN="${OUTPUT_BIN}.exe"
+            fi
+            echo "Building binary for $OS/$ARCH..."
+            GOARCH=$ARCH GOOS=$OS CGO_ENABLED=0 go build \
+                  -ldflags="-w -X main.VERSION=$VERSION" \
```

Comment:
Is there a specific reason for adding the `-w` linkflag to the build command?

Reply:
no, the whole build process is refer to https://github.com/rancher/cli/blob/v1.6/scripts/build